code/game/mecha/combat/tank.dm

```diff
@@ -0,0 +1,135 @@
+#define NOMINAL    0
+#define FIRSTRUN   1
+#define POWER      2
+#define DAMAGE     3
+#define IMAGE      4
+#define WEAPONDOWN 5
+
+/obj/mecha/combat/tank
```

Comment:
I don't know if this is a valid complaint considering that this will be an adminbus only option, however I'm a little concerned with the balance compared to other mechs.

It absorbs more laser weapon than a gygax, it's maximum temperature is higher than a gygax, it has significantly more health than a gygax (800 vs 300).